src/acceptance-test/java/org/zalando/nakadi/repository/db/TimelineDbRepositoryTest.java

```diff
@@ -116,20 +88,46 @@ public void testListTimelinesOrdered()
     }
 
     @Test
-    public void testTimelineDeleted()
-            throws InternalNakadiException, DuplicatedEventTypeNameException, DuplicatedStorageIdException {
-        final Storage storage = sRepository.createStorage(
-                StorageDbRepositoryTest.createStorage("default", "localhost", 8181, "test", "path"));
-        final EventType testEt = eRepository.saveEventType(TestUtils.buildDefaultEventType());
-        final Timeline t1 = tRepository.createTimeline(createTimeline(
-                storage, UUID.randomUUID(), 1, "test_topic", testEt.getName(),
-                new Date(), null, null, null));
+    public void testTimelineDeleted() {
+        final Timeline t1 = insertTimelineWithOrder(1);
         Assert.assertEquals(1, tRepository.listTimelinesOrdered(testEt.getName()).size());
         tRepository.deleteTimeline(t1.getId());
         Assert.assertEquals(0, tRepository.listTimelinesOrdered(testEt.getName()).size());
     }
 
-    private static Timeline createTimeline(
+    @Test
+    public void testGetExpiredTimelines() {
+        final DateTime now = new DateTime();
+        final DateTime tomorrow = now.plusDays(1);
+        final DateTime yesterday = now.minusDays(1);
+        final DateTime twoDaysAgo = now.minusDays(2);
+
+        insertTimeline(tomorrow.toDate(), true, 0);
+        insertTimeline(tomorrow.toDate(), false, 1);
+        insertTimeline(yesterday.toDate(), true, 2);
+        final Timeline t4 = insertTimeline(yesterday.toDate(), false, 3);
+        final Timeline t5 = insertTimeline(twoDaysAgo.toDate(), false, 4);
+
+        final List<Timeline> expiredTimelines = tRepository.getExpiredTimelines();
+
+        Assert.assertEquals(2, expiredTimelines.size());
```

Comment:
Good point 👍 
I now added explicit check for NULL and extended test https://github.com/zalando/nakadi/pull/611/commits/b0f8b6b49c16ad5b4f6a33e9599757faaa6d0172